### PR TITLE
Update readme according to issue #2

### LINKED ISSUE #2: Update README
- [x] add links to navigate LICENSE
      ex:- [LICENSE](https://github.com/MadhushaPrasad/vue-analog-clock/blob/main/LICENSE)
- [x] Add a contribution section.
- [x] Highlight keywords
- [x] Check spelling Mistakes and fix them.
- [x] Add an NPM package link. here is the link [https://www.npmjs.com/package/vue3-analog-clock]
- [x] Add contribution section
       ex: -  
## 🌱 Contributors </br>
<a href="https://github.com/MadhushaPrasad/vue-analog-clock/graphs/contributors">
  
</a>
</br>

Please replace the above code at the bottom of the readme file.

### PR DESCRIPTION
**Pull request by martincarapia**
I threw the spelling through Grammarly. This merge closes #2

### CODE REVIEW

**Comment by MadhushaPrasad** [comment]
I need to add more sections to the readme file. Can you do it? @martincarapia 

- [x] Add a contribution section.

File: README.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ app.mount('#app')
 
 ![ezgif-1-3d019c9ad5](https://github.com/MadhushaPrasad/vue-analog-clock/assets/50085447/fe6c60e1-a631-42cc-9d93-5bd1bc7e0791)
 
+### Contributors
 
-### Keywords
-Vue, Vue 3, Analog Clock, Clock Component, Vue Component
```
Comment (MadhushaPrasad) [inline README.md:47]: Don't remove these keywords. You need to bold it.

File: README.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ app.mount('#app')
...
+- [Madhusha Prasad](https://github.com/MadhushaPrasad)
```
Comment (MadhushaPrasad) [inline README.md]: Please replace the code that I commented on in the issue. #2

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -42,7 +43,7 @@ app.mount('#app')
 
 ![ezgif-1-3d019c9ad5](https://github.com/MadhushaPrasad/vue-analog-clock/assets/50085447/fe6c60e1-a631-42cc-9d93-5bd1bc7e0791)
 
-### Contributors
+# 🌱 Contributors
```
Comment (MadhushaPrasad) [inline README.md]: ## 🌱 Contributors </br>
<a href="https://github.com/MadhushaPrasad/vue-analog-clock/graphs/contributors">
  
</a>
</br>


Please replace this code

File: README.md [hunk 4]
```diff
@@ -13,6 +13,7 @@ This package provides a <mark>Vue 3</mark> component to display an <mark>analog
 ```bash
 npm install vue3-analog-clock
 ```
```
Comment (MadhushaPrasad) [inline README.md:15]: Add more space here, and this link needs to be opened in a new tab.

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]
Screenshot 2024-04-04 at 20 42 16 

Don't remove section. You need to bold (Vue, Vue 3, Analog Clock, Clock Component, Vue Component).

**Comment by martincarapia** [comment]
So you want the keywords bolded, not highlighted, correct?

**Comment by MadhushaPrasad** [comment]
> So you want the keywords bolded, not highlighted, correct?

yah, do bold

**Comment by MadhushaPrasad** [comment]
Screenshot 2024-04-04 at 20 46 38


after fixing those list items. Feel free to check that related task checkbox.

**Comment by martincarapia** [comment]
It doesn't let me check off the tasklist.

**Comment by martincarapia** [comment]
> > So you want the keywords bolded, not highlighted, correct?
> 
> yah, do bold

Alright, everything should be good.

**Comment by MadhushaPrasad** [comment]
> It doesn't let me check off the tasklist.

Go to this issue and do it there. #2

**Comment by martincarapia** [comment]
I'm pretty sure since you made the checklist you have to check it off. I could be wrong though

**Comment by MadhushaPrasad** [comment]
> I'm pretty sure since you made the checklist you have to check it off. I could be wrong though

ok then i will check it off

**Comment by MadhushaPrasad** [comment]
Screenshot 2024-04-04 at 21 12 22

remove this extra spaces
@martincarapia

**Review by MadhushaPrasad — APPROVED** [review]
All is fine now.
Thanks for your valuable contribution! Your effort is greatly appreciated.

**Pull request merged**